controllers/storagecluster/prometheus/localcephrules.yaml

```diff
@@ -278,6 +278,17 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: CephMonLowNumber
+      annotations:
+        description: The number of node failure zones available allow to increase the number of Ceph monitors in order to improve cluster resilience.
+        message: The current number of monitors can be increasedin order to improve cluster resilience
```

Comment:
Maybe better to mention, `ceph monitors`?
`Ceph monitors count can be increased to improve cluster resiliency`